.github/workflows/notion.yml

```
@@ -0,0 +1,101 @@
+            "properties": {
+              "Status (Backend)": {
+                "status": {
+                  "name": "${{ env.TASK_STATUS }}"
+                }
+              }
+            }
+          }'
+
+      - name: Post warning comment on PR
+        if: env.HAS_PBI == 'true' && env.TASK_FOUND == 'false'
+        uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)